src/Qir/Controller/Tests.QirController/ControllerTests.cs

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
```

Reply:
Nit: the header we are using now removed the "All rights reserved." part. Consider changing it in the other new files.